package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "vite-elm-template",
   "author": "Lindsay Wardell",
-  "description": "A staring template for building Elm apps with Vite",
+  "description": "A default template for building Elm applications using Vite.",
```

Comment:
Typo: staring -> star<strong>t</strong>ing

But I synced the description with the repo description and the README instead.